leafsmart-front-end/src/components/Events.js

```diff
@@ -48,28 +48,29 @@ const Events = (props) => {
     );
   });
 
-  return (
-    <section className="event-widget">
-      <div class="flex mt-6 mb-6 p-1 bg-white overflow-y-auto h-[43rem]">
-        <table className="w-fit bg-Isabelline border border-collapse">
-          <thead class="text-xs uppercase">
-            <tr>
-              <th class="border-collapse bg-Isabelline text-left px-8 py-1">
-                Event Details
-              </th>
-              <th class="border-collapse bg-Isabelline text-left px-8 py-1">
-                Event Date
-              </th>
-              <th class="border-collapse bg-Isabelline text-left px-8 py-1">
-                Get Tickets!
-              </th>
-            </tr>
-          </thead>
-          <tbody>{eventsArr}</tbody>
-        </table>
+
+return (
+  <section className="event-widget">
+    <div className="flex flex-col w-2/3 h-screen">
```

Comment:
The only thing I'd suggest tweaking are a couple classes here to get things centred! Looks like the event-widget class name's been phased out by tailwind in full now, so we should be safe to remove it. I tinkered and these classes seem to centre it nicely!: 
![image](https://user-images.githubusercontent.com/83998622/183748205-49ff2d68-edbf-4d66-9cf7-23769529efbd.png)